openshift/templates/deploy.dc.yaml

```diff
@@ -0,0 +1,119 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: ${NAME}
+  annotations:
+    openshift.io/display-name: "wps-api"
+    description: "Wildfire Predictive Services - APIent"
+    openshift.io/long-description: "Wildfire Predictive Services - Web Client"
+    tags: "wps,wps-api"
+    iconClass: icon-js
+    openshift.io/provider-display-name: "Government of British Columbia"
+    openshift.io/documentation-url: "https://github.com/bcgov/wps-api"
+    openshift.io/support-url: "https://github.com/bcgov/wps-api"
+labels:
+  app.kubernetes.io/part-of: "${NAME}"
+  app: ${NAME}-${SUFFIX}
+parameters:
+  - name: NAME
+    description: Module name
+    value: wps-api
+  - name: SUFFIX
+    description: Deployment suffix, e.g. pr-###
+    required: true
+  - name: DOCKER_REGISTRY
+    value: docker-registry.default.svc:5000
+  - name: PROJECT_TOOLS
+    value: auzhsi-tools
+  - name: CPU_REQUEST
+    description: Requested CPU
+    value: 1000m
```

Comment:
We probably don't actually need this much? I had a quick look at the load - and I bet we can get away with 1/4 of that. Might change later when we hit the real API but for now - we don't need that much. 250m?

Reply:
We can fine tune later, but otherwise done.